recipes/singlem/meta.yaml

```diff
@@ -16,43 +16,48 @@ build:
   run_exports:
     - {{ pin_subpackage('singlem', max_pin="x.x") }}
 
+# Requirements can be copied from the environment.yml file at https://github.com/wwood/singlem/blob/main/admin/environment.yml
 requirements:
   host:
     - pip
-    - python >=3.7
-    - nose >=1.0
+    - python >=3.12.3
+    - setuptools
   run:
-    - python >=3.7
-    - diamond >=2.1.7
-    - biopython =1.84
+    - biopython =1.85
+    - bird_tool_utils_python =0.4.1
+    - cd-hit =4.8.1
+    - diamond =2.1.10
+    - expressbetadiversity =1.0.10
+    - extern =0.4.1
```

Comment:
please do not use strict pins here, this will make your package incompatible with many environments